.travis.yml

```diff
@@ -42,6 +42,7 @@ before_script:
 
 script:
   - export PATH=$HOME/.cargo/bin:$PATH
+  - make emulation-setup
```

Comment:
one downside of this is that we lose fail-fast behavior for Travis when the error is something simple like a formatting error that would be caught in the first step. I only think this is okay because we have Github Actions now which should fail fast instead.